content/client-server-api/_index.md

```diff
@@ -970,6 +980,129 @@ in the registration process that their token has expired.
 
 {{% http-api spec="client-server" api="registration_tokens" %}}
 
+##### Terms of service at registration
+
+{{% added-in v="1.11" %}}
+
+| Type                     | Description                                                           |
+|--------------------------|-----------------------------------------------------------------------|
+| `m.login.terms`          | Authentication requires the user to accept a set of terms of service. |
```

Comment:
```suggestion
| `m.login.terms`          | Authentication requires the user to accept a set of terms of policy documents. |
```

to be consistent with [the paragraph below the note](https://github.com/matrix-org/matrix-spec/pull/1812/files#diff-c82b624572233237a687eb9baaf76eafd820da0b362057ed0851593343a37d3bR997)

Reply:
Applied a slightly different version.